osm/src/main/groovy/org/orbisgis/orbisprocess/geoclimate/osm/WorkflowOSM.groovy

```diff
@@ -115,7 +115,7 @@ def ProcessingChain = GroovyProcessManager.load(PC)
  */
 create {
     title "Create all Geoindicators from OSM data"
-    id "workflow"
+    id "Workflow"
```

Comment:
I think it will be better to set the the first letter in lower for processing name